runner/local/output.go

```diff
@@ -60,6 +65,13 @@ func (o *localOutput) Close() error {
 	return o.f.Close()
 }
 
+func (o *localOutput) Register(namespace, name string) (uri string) {
+	if o.handler != nil {
+		o.uri = o.handler.AddResource(namespace, name, o.AsFile())
```

Comment:
Why do we need to add the resource? It seems like it would just end up in the temp dir and so the link would work...